frontend/public/style/_overrides.scss

```diff
@@ -319,3 +319,7 @@ ul.pf-c-tree-view__list {
 .pf-c-skip-to-content {
   position: absolute !important;
 }
+
+.pf-topology-control-bar {
+  left: var(--pf-global--spacer--sm) !important;
+}
```

Comment:
This style should be in the Topology package. You can add it in `TopologyFilterBar.scss `.

Reply:
Found that the `TopologyFilterBar.scss` is imported in `TopologyFilterBar.tsx` file.  
But the control bar is not defined in that file, instead, the control bar is defined in `TopologyControlbar.tsx`.  
In `TopologyControlbar.tsx` file, `TopologyFilterBar.scss`  is not imported.  

So, if I mention this in the `TopologyFilterBar.scss` file, it shouldn't work.

Reply:
Found that the `TopologyFilterBar.scss` is imported in `TopologyFilterBar.tsx` file.  
But the control bar is not defined in that file, instead, the control bar is defined in `TopologyControlbar.tsx`.  
In `TopologyControlbar.tsx` file, `TopologyFilterBar.scss`  is not imported.  

So, if I mention this in the `TopologyFilterBar.scss` file, it shouldn't work.

Reply:
If that is the case, then create a `TopologyControlbar.scss` and import it in `TopologyControlbar.tsx`.  It will make the topology style independent of other packages.

Reply:
Thanks.  
Will update it :)